.github/2.0/cookbooks/1vs2.md

```diff
@@ -61,6 +61,29 @@ Benchmark #1: python -c "from jina import Document, Flow, Executor"
 
 </details>
 
-1. Keyword 3
-1. Keyword 4
-1. Keyword 5
\ No newline at end of file
+## 3. 2.0 Gives YOU the Power
+
+Jina 1.0 was a complex beast with lots of boilerplate code and not so much transparency. In 2.0 we're putting the power back in your hands. Want to write your own `.logger`, `.save()`, `.load()` or context manager interface for your Executor? Now that power is yours.
+
+## 4. 2.0 is More Pythonic
+
+In Jina 2.0 Documents and DocumentArrays are first-class citizens and fully Pythonic data types. With no more Drivers, you can manipulate Documents directly with all the power that entails:
```

Comment:
Perhaps: 

We embrace explicit above implicit: No more Drivers. You can manipulate Documents in custom Executors directly with all the power that entails:

```python
from jina import Document, DocumentArray, Executor, requests

class MyExecutor(Executor):
    @requests
    def add_world(self, docs: DocumentArray):
        for doc in docs:
            doc.text += 'world'
```